tools/clean.sh

```diff
@@ -25,3 +28,13 @@ clean_dir ./IntegrationTests/Pods
 echo_warn "Cleaning local xcconfigs"
 rm -vf ./xcconfigs/Base.ci.local.xcconfig
 rm -vf ./xcconfigs/Base.dev.local.xcconfig
+
+ENV_DEV="dev"
+ENV_CI="ci"
+
+if [[ "$ENV" == "ci" ]]; then
```

Comment:
nit/ This works rather by coincidence than explicit check, because this param is made for `repo-setup` command in `make clean repo-setup ENV=ci`.

More explicit is to check `$CI` env which is default for GitLab runner:
```suggestion
if [ "$CI" = "true" ]; then
```